chain/ethereum/src/block_ingestor.rs

```diff
@@ -1,9 +1,35 @@
+use std::collections::HashMap;
 use std::time::Duration;
 use std::time::Instant;
 
 use graph::prelude::*;
 use web3::types::*;
 
+pub struct BlockIngestorMetrics {
+    pub block_number: Box<GaugeVec>,
+}
+
+impl BlockIngestorMetrics {
+    pub fn new<M: MetricsRegistry>(registry: Arc<M>) -> Self {
+        Self {
+            block_number: registry
+                .new_gauge_vec(
+                    String::from("ethereum_block_number"),
```

Comment:
We've been calling this the chain head in graph-node. Let's stick to that and call this `ethereum_chain_head_number`.